shell/browser/api/electron_api_session.h

```diff
@@ -179,6 +208,19 @@ class Session : public gin::Wrappable<Session>,
   v8::Global<v8::Value> service_worker_context_;
   v8::Global<v8::Value> web_request_;
 
+  bool has_emitted_chrome_tabs_get_warning_ = false;
+  bool has_emitted_active_tab_warning_ = false;
```

Comment:
These should probably be behind the `ENABLE_ELECTRON_EXTENSIONS` buildflag below.